### DISCUSSION TITLE
Localization sync from tx

### DISCUSSION THREAD

**Discussion by NorwayFun** [sphinx-doc/sphinx#12887] [Q&A]
Hello,
I see localization files not being synced for sphinx and subcomponnents for 6 years at least. 
can someone do it, please?

**Comment by chrisjsewell** [comment]
heya, these are all synced automatically in https://github.com/sphinx-doc/sphinx/blob/master/.github/workflows/transifex.yml and https://github.com/sphinx-doc/sphinx-doc-translations/blob/master/.github/workflows/main.yml,
so you would need to be more specific about what you mean

**Reply by NorwayFun** [reply]
They're not for example here: https://github.com/sphinx-doc/sphinxcontrib-devhelp/tree/master/sphinxcontrib/devhelp/locales
